qa/suites/rgw/sts/tasks/ststests.yaml

```diff
@@ -1,8 +1,9 @@
 tasks:
 - s3tests:
     client.0:
-      extra_attrs: ['sts_test']
-      force-branch: master
+      git remote: git://github.com/TRYTOBE8TME/
```

Comment:
Same goes for this git remote.

Reply:
I've scheduled a teuthology run to check the new added test and because of this I've added git remote. I'll remove that after checking